app/src/main/res/font/pixelete.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<font-family xmlns:android="http://schemas.android.com/apk/res/android">
+    <font
+        android:font="@font/sf_pixelate_shaded_bold"
+        android:fontStyle="normal"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/worker8/Pixels/blob/43b06fc24f1ddcb819de2e20dacc4fe82bd270b7/app/src/main/res/font/pixelete.xml#L5"></span>Attribute <code>fontStyle</code> is only used in API level 26 and higher (current min is 21)</td>
    </tr>
  </tbody>
</table>